src/main/java/taxi/service/impl/AuthenticationServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package taxi.service.impl;
+
+import java.util.Optional;
+import taxi.dao.DriverDao;
+import taxi.exception.AuthenticationException;
+import taxi.lib.Inject;
+import taxi.lib.Service;
+import taxi.model.Driver;
+import taxi.service.AuthenticationService;
+
+@Service
+public class AuthenticationServiceImpl implements AuthenticationService {
+    @Inject
+    private DriverDao driverDao;
+
+    @Override
+    public Driver login(String login, String password) throws AuthenticationException {
+        Optional<Driver> driver = driverDao.findDriverByLogin(login);
+        if (driver.isEmpty()) {
+            throw new AuthenticationException("Login or password is incorrect");
+        }
+        if (driver.get().getPassword().equals(password)) {
+            return driver.get();
+        }
+        return null;
```

Comment:
```suggestion
        throw new AuthenticationException("Login or password is incorrect");
```
if password don't match lets give user corresponding message(through error throw) instead of returning null